cfgov/jinja2/v1/_includes/molecules/expandable.html

```diff
@@ -77,41 +74,5 @@
 </div>
 {% endmacro %}
 
-{% if not value %}
-    {% set value = [
-            {
-                'label': 'Default Expandable',
-                'paragraph': lipsum(1, false, 10, 225),
-                'is_bordered': false,
-                'is_midtone': false,
-                'is_expanded': false
-            },
-            {
-                'label': 'Bordered Expandable',
-                'paragraph': lipsum(1, false, 10, 225),
-                'is_bordered': true,
-                'is_midtone': false,
-                'is_expanded': false
-            },
-            {
-                'label': 'Midtone Expandable',
-                'paragraph': lipsum(1, false, 10, 225),
-                'is_bordered': true,
-                'is_midtone': true,
-                'is_expanded': false
-            },
-            {
-                'label': 'Midtone, Bordered, Expandable - Expanded',
-                'paragraph': lipsum(1, false, 10, 225),
-                'is_bordered': true,
-                'is_midtone': true,
-                'is_expanded': true
-            },
-        ]
-    %}
-    {% for exp in value %}
-        {{ expandable(exp) }}
-    {% endfor %}
-{% else %}
-    {{ expandable(value) }}
-{% endif %}
+
```

Comment:
need to add an if conditional 
```
{% if value %}
    {{ expandable(value) }}
{% endif %}
```
or it breaks pages that aren't currently using expandables due to imports e.g. Browse basic template: `{% import 'molecules/expandable.html' as expandable %}`